docs/updating/updating_to_3.3.md

```diff
@@ -167,7 +168,45 @@ Check the Location ID of the "Components" Content item and set it as a value of
 If you are upgrading between [[= product_name_com =]] versions,
 add the `content/read` Policy with the Owner Limitation set to `self` to the "Ecommerce registered users" Role.
 
-### Updating to 3.3.2
+## Finish the update
+
+Finish the update by running the following commands:
+
+``` bash
+php bin/console ibexa:graphql:generate-schema
+composer run post-install-cmd
+```
+
+## Update to v3.3.x
+
+To update from one v3.3 patch version to another (for example, from v3.3.1 to v3.3.2), run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:3.3.2 -W
+    composer recipes:install ibexa/content --force
```

Comment:
IMHO it's best to always add `-v` flag to increase verbosity:
`composer recipes:install ibexa/content --force -v`
(applies to other examples as well)